demisto_sdk/commands/content_graph/strict_objects/integration.py

```diff
@@ -40,7 +41,7 @@
 
 class _Configuration(BaseStrictModel):
     display: Optional[str] = None
-    section: Optional[str] = None
+    section: Optional[str]
```

Comment:
why did you remove the assignment?
The convention is None where the filed is Optional